### PR TITLE
Users full name instead of ID at comments

### PR DESCRIPTION
**Pull request by te4-david-petersson**
Fixed issue #57 
Users full name is now shown instead of their ID at comments

### CODE REVIEW

File: backend/server.rb [hunk 1]
```diff
@@ -101,11 +101,7 @@ def authenticated?
   get '/api/v1/users/:id' do
     if authenticated?
       user = Users.get(params['id'])
```
Comment (TE4-arvid-nilber) [inline backend/server.rb]: Maybe you could add some more security here, maybe you could check if the logged in user has the same teacher_id as the id of the user that's requested?

File: frontend/src/components/LogCommentComponent.vue [hunk 2]
```diff
@@ -16,19 +14,20 @@ import UsersModule from "@/store/modules/UsersModule";
 
 @Component
 export default class LogCommentComponent extends Vue {
-    @Prop()
-    private comment!: any;
-    @Prop()
-    private log!: any;
+  @Prop()
+  private comment!: any;
+  //hola
```
Comment (TE4-arvid-nilber) [inline frontend/src/components/LogCommentComponent.vue]: Why?

Comment (te4-david-petersson) [reply frontend/src/components/LogCommentComponent.vue]: dunno, i fix

File: frontend/src/store/modules/UsersModule.ts [hunk 3]
```diff
@@ -120,11 +124,19 @@ class UsersModule extends VuexModule {
     axios
       .get(`http://localhost:9292/api/v1/${type}`)
       .then(response => {
+<<<<<<< HEAD
```
Comment (itggot-linus-styren) [inline frontend/src/store/modules/UsersModule.ts]: what...

Comment (arvidnilber) [reply frontend/src/store/modules/UsersModule.ts]: perfectly fine

Comment (itggot-linus-styren) [reply frontend/src/store/modules/UsersModule.ts]: but... 😔🤔🙁😕

File: backend/models/users.rb [hunk 4]
```diff
@@ -54,7 +54,9 @@ def self.not_teachers_students(teacher_id)
     # Login logic, returns a JWT token to the user.
     def self.login(user_form) 
         @db.results_as_hash = true
+
         user = @db.execute('SELECT * FROM users WHERE email = ?', user_form['email']).first
+        p user
```
Comment (TE4-Mattis-Abrahamsson) [inline backend/models/users.rb]: How about removing this?

**Pull request merged**